test/ci/kokoro/linux/run_integ_tests.sh

```diff
@@ -76,6 +79,8 @@ init_configs
 init_python
 update_submodules
 
+# Check that we're using the correct config
+gsutil version -l
```

Comment:
This will invoke the version of gsutil that comes with the Kokoro host environment (via the Cloud SDK).  I think you probably want this to be `python "$GSUTIL_ENTRYPOINT" version -l`.

Reply:
Oops, good catch! Will update to use the repo's gsutil.